module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -141,6 +141,16 @@ protected function snooze($secs = 1)
         usleep(1000000 * $secs * $snoozeMultiplier);
     }
 
+    /**
+     * Get the default timeout in milliseconds
+     *
+     * @return int
+     */
+    protected function getDefaultTimeout(): int
+    {
+        return intval(getenv('VUFIND_DEFAULT_TIMEOUT') ?: self::DEFAULT_TIMEOUT);
```

Comment:
Maybe VUFIND_DEFAULT_TEST_TIMEOUT or VUFIND_DEFAULT_MINK_TIMEOUT for clarity?

Reply:
I chose VUFIND_DEFAULT_TEST_TIMEOUT (trying to keep component names out of general settings). :)